### ISSUE TITLE
Trader ships with red outlines at the beginning of Level 2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#375]
```
Game version:
0.94.364

Operating System:
Windows 7 x86

Java runtime version:
1.6.0.31

Installed using the Launcher? (yes, no)
yes

Game language (en, hu):
hu

What steps will reproduce the problem?
Get promoted to captain. (I think level 2 is captain)

What is the expected output? What do you see instead?
As soon as you get promoted, some of the trader ships will have red outlines 
around them, just like when there's a virus infection, and you have to prevent 
them from reaching a planet. But this time, there's no real warning or incoming 
message mentioning this. Later on, when you prevent the smuggler from reaching 
San Sterling, I think the problem disappears.


```

Original issue reported on code.google.com by `sosetala...@gmail.com` on 25 Apr 2012 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
Yes a small indicator bug affecting traders flying to San Sterling before 
mission 9 is actually running.
```

Original comment by `akarn...@gmail.com` on 25 Apr 2012 at 9:49
- Changed state: **Accepted**
- Added labels: **Milestone-0.94.400**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.94.365
```

Original comment by `akarn...@gmail.com` on 29 Apr 2012 at 10:37
- Changed state: **SVN**

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 30 Apr 2012 at 7:08
- Changed state: **Fixed**
